<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xSeqChildDlgClass</class>
 <widget class="QDialog" name="xSeqChildDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xSeqChildDlg</string>
  </property>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnContinue">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
